src/test/scala/nl.knaw.dans.easy.validatebag/XmlValidatorSpec.scala

```diff
@@ -25,10 +25,10 @@ import org.scalatest.{ FlatSpec, Matchers }
 
 import scala.util._
 
-class XmlValidatorSpec extends FlatSpec with Matchers {
+class XmlValidatorSpec extends TestSupportFixture with Matchers {
```

Comment:
As you use `TestSupportFixture` now, you also don't need `Matchers` anymore.
Also don't forget to remove the unused imports!